tests/topotests/bgp_evpn_rt5/r1/frr.conf

```diff
@@ -23,14 +24,10 @@ router bgp 65000
  no bgp ebgp-requires-policy
  neighbor 192.168.100.41 remote-as 65000
  neighbor 192.168.100.41 capability extended-nexthop
- neighbor 192.168.100.61 remote-as 65500
- neighbor 192.168.100.61 capability extended-nexthop
  !
```

Comment:
wrong assumption. this has been introduced to test the match evp vni functionality.
https://github.com/FRRouting/frr/commit/0d5bd461af9f

Reply:
https://github.com/FRRouting/frr/commit/0d5bd461af9f was not correct. It added r3 but was working even without r3.

I have added "tests: bgp_evpn_rt5 test match evpn vni" to properly test the feature